src/components/Tasks/TaskList.tsx

```diff
@@ -31,9 +38,26 @@ interface PropsType {
 
 export default React.memo(function TaskList(props: PropsType) {
   const [showCompleted, setShowCompleted] = React.useState(false);
-  const entries = props.entries.filter((x) => showCompleted || !x.finished);
+  const settings = useSelector((state: StoreState) => state.settings.taskSettings);
+  const { filterBy } = settings;
+  const theme = useTheme();
+
+  const potentialEntries = props.entries.filter((x) => showCompleted || !x.finished);
+  let entries;
+  if (filterBy?.startsWith('tag')) {
+    const tag = filterBy.slice(4);
```

Comment:
In continuation to my comment about making the delimiter `:`, the above should probably be changed to `startsWith('tag:')` (now there's a bug if we ever add another word that starts with tag).

Having the magic number 4 here feels a bit wrong to me tbh. `filterBy.slice(filterBy.indexOf(':') + 1)` is ugly as hell, though maybe better? Or maybe even:
```
const tagPrefix = 'tag:';
if (filterBy?.startsWith(tagPrefix)) {
    const tag = filterBy.slice(tagPrefix.length);
```